docs/pages/guides/using-sentry.md

```diff
@@ -68,7 +70,11 @@ Sentry.Native.*
 Sentry.Browser.*
 ```
 
-- Open **app.json** and add a `postPublish hook`:
+### Step 3: App Config
+
+#### Configure your `postPublish` hook
+
+Add `expo.hooks` to your project's `app.json` (or `app.config`) file:
```

Comment:
Should this be `app.config.js` , not just `app.config` ?

---
**Sema Reaction:** :question: I have a question

Reply:
thanks @gregfenton ! https://github.com/expo/expo/commit/2b728c42a42b25223c723d0107f80e7cbfc3e5e1